CHANGELOG.md

```diff
@@ -37,19 +37,21 @@ Early Access Packages
 | Package                                    | Released version   | Target version |
 |--------------------------------------------|--------------------|----------------|
 | ContingentClaims.Valuation                 | 0.2.0              | 0.2.1          |
-| Daml.Finance.Instrument.Bond               | 0.2.1              | 0.2.2          |
-| Daml.Finance.Instrument.Equity             | 0.2.1              | 0.2.2          |
-| Daml.Finance.Instrument.Option             | 0.1.0              | 0.1.1          |
-| Daml.Finance.Instrument.Swap               | 0.2.1              | 0.2.2          |
-| Daml.Finance.Interface.Instrument.Bond     | 0.2.1              | 0.2.1          |
-| Daml.Finance.Interface.Instrument.Equity   | 0.2.0              | 0.2.1          |
-| Daml.Finance.Interface.Instrument.Option   | 0.1.0              | 0.1.0          |
-| Daml.Finance.Interface.Instrument.Swap     | 0.2.1              | 0.2.2          |
+| Daml.Finance.Instrument.Bond               | 0.2.1              | 0.3.0          |
+| Daml.Finance.Instrument.Equity             | 0.2.1              | 0.3.0          |
+| Daml.Finance.Instrument.Option             | 0.1.0              | 0.2.0          |
+| Daml.Finance.Instrument.Swap               | 0.2.1              | 0.3.0          |
+| Daml.Finance.Interface.Instrument.Bond     | 0.2.1              | 0.3.0          |
+| Daml.Finance.Interface.Instrument.Equity   | 0.2.0              | 0.3.0          |
+| Daml.Finance.Interface.Instrument.Option   | 0.1.0              | 0.2.0          |
+| Daml.Finance.Interface.Instrument.Swap     | 0.2.1              | 0.3.0          |
 
 ## Pending changes
 
 #### ContingentClaims.Core
 
+- Dependencies update
```

Comment:
SDK update?

Reply:
yes, actually we could leave it at SDK 2.5.0. The only change would be the bump of the SDK version, so probably not worth releasing that, right?

Reply:
It needs to be released anyway given the changes to the `Or` node